.github/workflows/i18n-push.yml

```diff
@@ -26,32 +26,19 @@ jobs:
       # install dependencies
       - name: Install dependencies
         run: |
-          pip install -e ".[all]"  
-
-      # install transifex client
-      - name: transifex-client
-        run: |
-          pip install transifex-client
-
-      # store token
-      - name: store token
-        run: |
-          touch ~/.transifexrc
-          echo [https://www.transifex.com] >> ~/.transifexrc
-          echo api_hostname=https://api.transifex.com >> ~/.transifexrc
-          echo hostname=https://www.transifex.com >> ~/.transifexrc
-          echo password=${{ secrets.TRANSIFEX_TOKEN }} >> ~/.transifexrc
-          echo username=api  >> ~/.transifexrc
+          pip install -e ".[all]"
 
       # extract
       - name: extract_messages
         run: |
           python setup.py extract_messages
```

Comment:
would this command: still work?

Reply:
Thanks for the input. It should be changed to `python setup extract_messages`.